setup.py

```diff
@@ -32,11 +32,15 @@
 def get_requirements():
     packages = []
 
-    with open("./requirements.txt") as fd:
+    with open("./requirements.in") as fd:
         for line in fd.readlines():
             if not line or line.startswith('-i'):
                 continue
-            packages.append(line.split(';', 1)[0])
+            parsed_line = line.split(';', 1)[0].strip()
+            packages.append(parsed_line)
+
+    if sys.version_info.major == 3:
+        packages.remove("enum34")
```

Comment:
Not sure why is this needed, but OK. Can you perhaps add a comment (into setup.py) explaining the hack ?